graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,373 @@
+            Number of repetitions in each iteration of the iterative optimal
+            transport problem. I.e. maximum number of repetitions in each
+            "E-step".
+
+        optimal_transport_eps : float (default=0.1), optional
+            Tolerance for the each iteration of the iterative optimal transport
+            problem. I.e. tolerance of the whole "EM" algorithm.
+
+        iterative_num_reps : int (default=100), optional
+            Number of reps in each iteration of the iterative optimal transport
+            problem. I.e. maxumum number of total iterations the whole "EM"
+            algorithm.
+
+        initialization: string, {"2d" (default), "sign_flips", "custom"}, optional
+
+            - "2d"
+                uses 2^d different initiazlizations, where d is the dimension.
+                specifically, uses all possible matrices with all entries real
+                and diagonal entries having magnitude 1 and 0s everywehre else.
+                for example, for d=2, tries [[1, 0], [0, 1]], [[1, 0], [0,
+                -1]], [[-1, 0], [0, 1]], and [[-1, 0], [0, -1]]. picks the best
+                one based on the value of the objective function.
+            - 'sign_flips'
+                for the first dimension, if two datasets have medians with
+                varying signs, flips all signs along this dimension for the
+                first dataset. then initializes to an identity.
+            - "custom"
+                expects either an initial matrix Q or initial matrix P during
+                the use of fit or fit_transform. uses initial Q provided,
+                unless it is not provided. if not provided - uses initial P. if
+                neither is given initializes to Q = I.
+
+        initial_Q: np.ndarray, shape (d, d) or None, optional (default=None)
+            An initial guess for the alignment matrix, if such exists. Ignored
+            if initialization alignment is set to anything other than 'custom'.
+            If None - initializes using an initial guess for P.
+            If None, and P is also None - initializes Q to identity matrix.
+
+        initial_P: np.ndarray, shape (n, m) or None, optional (default=None)
+            Initial guess for the initial transport matrix.
+            Only matters if Q=None.
+
+        freeze_Y : boolean, optional (default=True)
+            Irrelevant in SeedlessProcrustes, as it always modifies only the
+            first dataset. Exists for compatibility with other align modules.
+
+    Attributes
+    ----------
+        Q_X : array, size (d, d)
+              final orthogonal matrix, used to modify X
+
+        Q_Y : array, size (d, d)
+              final orthogonal matrix, used to modify Y.
+              in SeedlessProcrustes Q_Y is always equal to identity I
+
+        P : array, size (n, m) where n and md are the sizes of two datasets
+            final matrix of optimal transports
```

Comment:
these should have trailing underscores as they are computed attributes

Reply:
👍

Reply:
fixed as a part of the API changes